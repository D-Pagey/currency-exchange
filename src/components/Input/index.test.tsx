import React from 'react';
import { render } from '@testing-library/react';
import { Input } from '.';

const props = {
    onChange: () => null,
    testId: '',
    type: '',
    value: 10,
};

describe('Input component', () => {
    it('should render', () => {
        const { container } = render(<Input {...props} />);
        expect(container.firstChild).toMatchSnapshot();
    });
});
