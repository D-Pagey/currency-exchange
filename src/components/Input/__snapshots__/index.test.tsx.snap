// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input component should render 1`] = `
<input
  class="sc-AxjAm fDCHeJ"
  data-testid=""
  inputmode="numeric"
  placeholder="undefined0"
  type="text"
  value="10"
/>
`;
